### ISSUE TITLE
Stuck with a bad Lottie version

### ISSUE THREAD

**Issue by eygraber** [airbnb/lottie-android#882]
I use Lottie version 2.5.5 in my app. I recently started targeting API 28, and [Lottie crashes](https://github.com/airbnb/lottie-android/issues/747) on devices running P. This was fixed in Lottie version 2.5.6, but that also introduced a change that [requires mindSdk >= 24](https://github.com/airbnb/lottie-android/issues/822) which I don't have, and so I can't build my app if I'm using Lottie 2.5.6. This was fixed in Lottie version 2.5.7 which [was not released properly on Maven](https://github.com/airbnb/lottie-android/issues/835) and so I can't use it.

In short, I have no way of using Lottie, because there is no version available that's compatible with Android P and minSdk < 24. I've had to disable any of our Lottie animations until this is fixed. Can you guys properly deploy 2.5.7, or something like that, so my PM will stop asking me when the animation is coming back?

**Comment by kofboy2000** [comment]
I have a same issue on P OS. May I know release(registration) schedule of 2.5.7 on Maven?

**Comment by jacek-marchwicki** [comment]
You can use the version from jitpack: https://jitpack.io/#airbnb/lottie-android/v2.5.7

**Comment by gpeal** [comment]
I just released 2.6.0-rc2
